functions/slack/app/request.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018, Google, LLC.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_slack_request]
+
+package slack
+
+import (
+	"fmt"
+)
+
+func makeSearchRequest(query string) (*Message, error) {
```

Comment:
In the context of the tutorial, I think it will be clear what this function is even if the region tag doesn't include the package declaration and imports. What do you think? Could we move this to the file where it's called from?

Same question with the other helper methods.